WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -229,32 +244,50 @@ void ScheduleRestartAutomatically(Wallet walletToStart, bool overridePlebStop)
 				return;
 			}
 
+#pragma warning disable CA2000 // Dispose objects before losing scope
+			var linkedCts = CancellationTokenSource.CreateLinkedTokenSource(stoppingToken);
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
Can't this be moved to be a property?

Reply:
It is created per task, so there are more of this.